src/Admin/WebspaceSettingAdmin.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Admin;
+
+use App\Entity\WebspaceSetting;
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Admin\View\ToolbarAction;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewBuilderFactoryInterface;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewCollection;
+use Sulu\Bundle\PageBundle\Admin\PageAdmin;
+use Sulu\Component\Security\Authorization\PermissionTypes;
+use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+use Sulu\Component\Webspace\Webspace;
+
+class WebspaceSettingAdmin extends Admin
+{
+    public const TAB_VIEW = 'app.webspace_settings';
+    public const FORM_VIEW = 'app.webspace_settings.form';
+    public const FORM_KEY = 'webspace_setting_details';
+
+
+    public function __construct(
+        private WebspaceManagerInterface $webspaceManager,
+        private ViewBuilderFactoryInterface $viewBuilderFactory,
+        private SecurityCheckerInterface $securityChecker
+    ) {
+    }
+
+    public function configureViews(ViewCollection $viewCollection): void
+    {
+        if ($this->hasSomeWebspaceSettingPermission()) {
+            $viewCollection->add(
+                // sulu will only load the existing entity if the path of the form includes an id attribute
+                $this->viewBuilderFactory->createResourceTabViewBuilder(static::TAB_VIEW, '/webspace-settings/:id')
+                    ->setResourceKey(WebspaceSetting::RESOURCE_KEY)
+                    ->setTabTitle('app.webspace_settings')
+                    ->setParent(PageAdmin::WEBSPACE_TABS_VIEW)
+                    ->setOption('routerAttributesToFormRequest', ['webspace'])
```

Comment:
This should be get part of the `createResourceTabViewBuilder` as `addRouterAttributesToFormRequest` when implemented

Reply:
> This should be get part of the `createResourceTabViewBuilder` as `addRouterAttributesToFormRequest` when implemented

Is there anything planed to get the webspace in createResourceTabViewBuilder?
THX for info

Reply:
The Builder is generally for all webspaces. To get the webspace in the endpoint which this view is for you need `$request->query->get('webspace');`. If you want to hide a tab you may could try use tabCondition but I'm not sure which data is available in the condition itself.

Reply:
Something is different in the admin build, so I run in a permission view error because I didn't get the webspace key with $request->query->get('webspace')
Is there any sulu js extension in the example, that is not loaded via bin/console sulu:admin:update-build?
THX